ui/src/userInput/showInputBox.ts

```diff
@@ -40,9 +44,14 @@ export async function showInputBox(context: IInternalActionContext, options: Inp
                     inputBox.enabled = true;
                     inputBox.busy = false;
                 }),
-                inputBox.onDidTriggerButton(_btn => {
-                    // Only back button is supported for now
-                    reject(new GoBackError());
+                inputBox.onDidTriggerButton(async btn => {
+                    if (btn === QuickInputButtons.Back) {
+                        reject(new GoBackError());
+                    } else if (btn === AzExtQuickInputButtons.LearnMore) {
+                        await openUrl(nonNullProp(options, 'learnMoreLink'));
+                        context.telemetry.properties.learnMoreStep = context.telemetry.properties.lastStep;
```

Comment:
Maybe add `learnMoreStep` here as well just to be consistent:
https://github.com/microsoft/vscode-azuretools/blob/df7af8b62562e7c25ba132d9aa3fc848cb6f2971/ui/src/userInput/showWarningMessage.ts#L34